packages/modal/addon/utils/event-handler.js

```diff
@@ -0,0 +1,15 @@
+import { isPresent } from '@ember/utils';
```

Comment:
I think we can get rid of this util if we set proper default values and check for querySelector's return value before adding event listeners.

Reply:
_BONUS POINTS_ if you could get rid of Ember specific libraries and make it a **pure** JS utility.